ONSdigital/eq-author
# eq-415 New Header/Button Styles

### What is the context of this PR?

[Trello](https://trello.com/c/9ycq3cOD/415-new-header)

- Implements design of new header
- Rewrote `Button` component with new styles
- Introduced `LinkButton` component for external, preview link
- Introduced `RouteButton` component for buttons that navigate internally
- Removed `IconButton` component, moving icon+text pattern into `IconText`
- Primary blue colour updated (more refactoring around colours required)

### How to review 

- Check header against designs
- Check buttons all work as expected
